src/main/java/commands/AddCommand.java

```diff
@@ -0,0 +1,37 @@
+package commands;
```

Comment:
You followed the java coding standards for your package statement. You did this for other package statements as well. Nice!